Gemfile.lock

```diff
@@ -62,18 +60,26 @@ GEM
     rexml (3.2.6)
     rouge (4.1.3)
     safe_yaml (1.0.5)
+<<<<<<< HEAD
+    sass-embedded (1.64.1-arm64-darwin)
+      google-protobuf (~> 3.23)
+=======
     sass-embedded (1.64.2)
       google-protobuf (~> 3.23)
       rake (>= 13.0.0)
+>>>>>>> 34d41e4c978d24d1abbaca6fa82a0a94623533e5
     terminal-table (3.0.2)
       unicode-display_width (>= 1.1.1, < 3)
     unicode-display_width (2.4.2)
     webrick (1.8.1)
 
 PLATFORMS
   arm64-darwin-22
+<<<<<<< HEAD
+=======
   x86_64-darwin-19
   x86_64-linux
+>>>>>>> 34d41e4c978d24d1abbaca6fa82a0a94623533e5
```

Comment:
Looks like accidental commit from merge conflict

Reply:
sorry about that, should be good to go now!